packages/time-picker/test/time-picker.test.js

```diff
@@ -89,14 +92,12 @@ describe('time-picker', () => {
       expect(comboBox.value).to.be.equal('');
     });
 
-    it('should restore the previous value in input field if input value is empty', () => {
-      comboBox.value = '12:00';
-      comboBox.value = '';
-      expect(timePicker.value).to.be.equal('');
-      setInputValue(comboBox, '');
-      enter(timePicker.inputElement);
+    it('should set empty value on outside click after clearing input value', () => {
```

Comment:
This test was added in https://github.com/vaadin/vaadin-time-picker/pull/104 and was intended to cover a fix for https://github.com/vaadin/vaadin-time-picker/issues/88. Updated the test to emulate sequence of steps described in that issue (clear the input value and click outside).